### DISCUSSION TITLE
Optimization of official website documents about streampark

### DISCUSSION THREAD

**Discussion by caicancai** [apache/streampark#2962] [General]
There are still many places that can be optimized in the streampark official document, which can be summarized and then evaluated and optimized together.

**Comment by caicancai** [comment]
Some parameter configurations do not correspond to the source code
<img width="470" alt="屏幕截图 2023-08-24 162552" src="https://github.com/apache/incubator-streampark/assets/77189278/a252664a-65a7-4ea4-b984-9ff5b1c11e2f">

**Reply by caicancai** [reply]
done

**Comment by caicancai** [comment]
# Streampark documentation V2

# 1. Configure parameter description
![屏幕截图 2023-09-10 142246](https://github.com/apache/incubator-streampark/assets/77189278/f0eeeada-00b4-4513-8aa4-04d4b9a1ad67)

For example, the current kubernetes_V2 module executing in streampark, flinkversion must be in the form of 1.17.1/1.17, if the 17.1 form may report an error

# 2. Image update
For previous StreamX style images, consider updating the StreamPark style

![image](https://github.com/apache/incubator-streampark/assets/77189278/5936584b-3020-458d-a419-9da96c85246e)
There are also some service configuration pictures that do not match the current interface

# 3. Complete description of the steps
For example, add LDAP configuration here, there are two steps here, in fact, you can add a third step here, and then use it as usual (directly start) or something, so as not to cause ambiguity to the user (such as the document is not finished)

![屏幕截图 2023-09-10 142858](https://github.com/apache/incubator-streampark/assets/77189278/dbe676fc-420b-462d-a630-4a1ac604b1be)

# 4. Heading style specification

# 5. K8S document and Hadoop document optimization
Now the Kubernetes documentation and Hadoop documentation are relatively simple and can be more detailed

For example, K8S documents can be described by Appliction, Session, and Cluster respectively, for example, to configure Session, you need to configure cluster first

I think it is good to make it clear about task publishing, task recovery, and task parameter configuration in the corresponding K8S document or Hadoop document, and there is no need to write a new document

# 6. Some Chinese documents do not match the English documents

![屏幕截图 2023-09-10 150822](https://github.com/apache/incubator-streampark/assets/77189278/bd99e25c-8244-430f-a308-8ccab75e433f)

Here I think it is better to follow the English document

# 7. Normalization of proper nouns

For example, Some proper nouns such as 火花图 do not have to be translated into Chinese

# 8.What is Streampark document optimization
The framework introduction adds a usage scenario description to let users understand what problems StreamPark can solve

**Comment by caicancai** [comment]
@wolfboys

**Reply by wolfboys** [reply]
Thank you for your collection and feedback. These areas have all been adjusted and optimized. 👍 BTW, The features of k8sV2 are still work in progress. The documentation can be prepared now. Our goal is for v2 to replace the implementation of v1, including the documentation. They will coexist for a period of time until v2 becomes stable.

**Reply by caicancai** [reply]
get

**Comment by Kick156** [comment]
I feel like we should add a versioning button to the document.
![image](https://github.com/apache/incubator-streampark/assets/107044617/d13ee9f7-c464-470b-bdc8-64f268c593cc)

**Reply by caicancai** [reply]
I think this is actually not necessary unless there are major changes between versions, and the number of streampark versions is indeed not many. There are more subsequent versions, so I think it can be considered. Of course, this is just my opinion. I'd love to hear your opinion, bro.

**Reply by Kick156** [reply]
Hi bro. Once I forgot how to do it when deploying a new version, and I wanted to see the documentation on the official website, but the documentation disappeared. I found it in the web project through GitHub during the discussion. So why can't it be solved by one-click version switching?
     Strictly speaking, Dinky, also as a streaming data development platform, has clear version management in its documentation, and it is not yet an Apache project, shouldn't we, as a project that entered the Apache incubator, do a better job in this regard.
     Of course, this is just my personal idea, and it is okay if you don't get it for the time being.

**Reply by wolfboys** [reply]
In versions prior to 2.0 of StreamX, We not necessary to provide documentation. We can offer multi-version documentation support for major releases. For example, 2.0.0 and the upcoming 2.2.

**Comment by caicancai** [comment]
Supplemental contributor documentation

**Reply by caicancai** [reply]
done
